modules/terraform-aws-ca-lambda/lambda_code/tls_cert/tls_cert.py

```diff
@@ -206,12 +207,22 @@ def create_ca_chain_response(project: str, env_name: str, root_ca_name: str, iss
     )
 
 
+def sns_notify_cert_issued(cert_json, sns_topic_arn):
+    keys_to_publish = ["CertificateInfo", "Base64Certificate", "Subject"]
+    response = publish_to_sns(cert_json, "Certificate Issued", sns_topic_arn, keys_to_publish)
+
+    assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
+    common_name = cert_json["CertificateInfo"]["CommonName"]
+    print(f"Certificate details for {common_name} published to SNS")
+
+
 def lambda_handler(event, context):  # pylint:disable=unused-argument,too-many-locals
     project = os.environ["PROJECT"]
     env_name = os.environ["ENVIRONMENT_NAME"]
     external_s3_bucket_name = os.environ["EXTERNAL_S3_BUCKET"]
     internal_s3_bucket_name = os.environ["INTERNAL_S3_BUCKET"]
     max_cert_lifetime = int(os.environ["MAX_CERT_LIFETIME"])
+    sns_topic_arn = os.environ["SNS_TOPIC_ARN"]
```

Comment:
Do we want to make `sns_topic_arn` optional?

Reply:
In my opinion, the code is kept simpler and more straightforward if we don't make it optional - and I can't think of a reason why anyone would object to having a SNS topic included with the CA